Fix telemetry typos

konstantin-msft opened the pull request:
- Remove `startPerformanceMeasuremeant` that is a duplicate of `startPerformanceMeasurement`.
- Rename `astsAsyncMeasurement` -> `atsAsyncMeasurement`.

konstantin-msft committed:
- Remove `startPerformanceMeasuremeant` that is a duplicate of `startPerformanceMeasurement`.
- Rename `astsAsyncMeasurement` -> `atsAsyncMeasurement`.

konstantin-msft committed:
Change files

tnorling reviewed (commented):
There was something we implemented to try both spellings when we rolled back the original change. Can we also revert that to only use the correct spelling?

The changes in [this PR](https://github.com/AzureAD/microsoft-authentication-library-for-js/pull/5589/files) are what I'm referring to

konstantin-msft committed:
- Remove the addressed comment.

konstantin-msft commented:
> There was something we implemented to try both spellings when we rolled back the original change. Can we also revert that to only use the correct spelling?
> 
> The changes in [this PR](https://github.com/AzureAD/microsoft-authentication-library-for-js/pull/5589/files) are what I'm referring to

I have addressed that already here https://github.com/AzureAD/microsoft-authentication-library-for-js/commit/b03430c5841e5caad33d0b928b160af9f1535708#diff-5c2e3cc078d85a3fa0eb4b4596ecee70cc03689d6ac7ab96c69b8628fd8e055fR233.

ghost commented:
:tada:`@azure/msal-browser@v3.0.0-alpha.0` has been released which incorporates this pull request.:tada:

Handy links:
* [Release Notes](https://github.com/AzureAD/microsoft-authentication-library-for-js/releases/tag/msal-browser-v3.0.0-alpha.0)
* [Official NPM Release](https://www.npmjs.com/package/@azure/msal-browser/v/3.0.0)
